CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 **Fixed**:
 
 - **decidim-core**: Fix notifications list when the resource is a User. [\#2687](https://github.com/decidim/decidim/pull/2687)
+- **decidim-core**: Fix mention parsing when there's more than one mention. [\2690](https://github.com/decidim/decidim/pull/2690)
```

Comment:
Perhaps, better if we change "parsing" for "rendering"?

Reply:
Oh, definitely.